src/ui/components/seekbar/useSlider.ts

```diff
@@ -21,8 +21,11 @@ export const useSlider = (): [number, boolean, React.Dispatch<React.SetStateActi
   useEffect(() => {
     if (!player) return;
     // Block time updates while scrubbing
-    if (isScrubbing) return;
-    const onTimeUpdate = () => {
+    const onTimeUpdate = (event: TimeUpdateEvent | SeekingEvent | SeekedEvent) => {
+      if (event.type === PlayerEventType.SEEKED) {
+        setIsScrubbing(false)
```

Comment:
This isn't right. The act of scrubbing over the seekbar doesn't "end" when the player finishes its current seek. For example:
1. The user starts dragging the seekbar, holding down the left mouse button
2. The player starts seeking, fires `seeking`
3. The player finishes seeking, fires `seeked`
4. The user drags the mouse further along the seekbar
5. The player starts a new seek, fires `seeking`
6. The user releases the left mouse button
7. The player finishes seeking, fires `seeked`

`isScrubbing` should remain `true` all the way through steps 1-6 (and maybe also 7?). With this code, it would switch to `false` too early after step 3.